Source/Generic/ECTestCase.m

```diff
@@ -719,19 +720,19 @@ - (BOOL)image:(NSBitmapImageRep*)image matchesReferenceImage:(NSBitmapImageRep*)
 	CGFloat pixelThreshold = [mergedProperties[@"pixelThreshold"] doubleValue];
 	NSSize maxSize = NSZeroSize;
 	if ([mergedProperties[@"maxSizeMatchesReference"] boolValue]) {
-		maxSize = reference.size;
+		maxSize = NSMakeSize(reference.pixelsWide, reference.pixelsHigh);
 	} else {
 		maxSize = NSMakeSize([mergedProperties[@"maxWidth"] doubleValue], [mergedProperties[@"maxHeight"] doubleValue]);
 	}
 
-	NSSize imageSize = image.size;
+	NSSize imageSize = NSMakeSize(image.pixelsWide, image.pixelsHigh);
 	if ((imageSize.width > maxSize.width) || (imageSize.height > maxSize.height))
 	{
 		NSLog(@"image looks a bit big: %@", NSStringFromSize(imageSize));
 		return NO;
 	}
 
-	NSSize referenceSize = reference.size;
+	NSSize referenceSize = NSMakeSize(reference.pixelsWide, reference.pixelsHigh);
```

Comment:
The pixelsWide and pixelsHigh properties is the image size in pixels, while the size property is in points and is impacted by a scale factor which can be obtained from the imported image metadata. We want to do a pixel by pixel comparison for the tests.

The thing that triggered this is some artboards have an @ symbol in their name which we generate the reference image file names from. The OS when reading these images gives the image a scale factor. This gets around that and for the future if we get any new documents that get added to the tests with artboard names that include an @ symbol in their name we won't have a repeat of the problem.